Homework1.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Inventorying Items</title>
+  <script src="https://code.jquery.com/jquery.min.js"></script>
+  <link href="https://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css" rel="stylesheet" type="text/css" />
+  <script src="https://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/js/bootstrap.min.js"></script>
+  <script src="https://code.jquery.com/jquery-3.1.0.js"></script>
+  <style>
+    body{
+      margin-left: 50px;
+      margin-top: 50px;
+    }
+    span{
+      margin-bottom: 50px;
+    }
+    div{
+      margin-bottom: 30px;
+      margin-top: 30px;
+    }
+  </style>
+</head>
+<body>
+  <span><h4>Name:</h4><input id="nameInput"></input></span><br>
+  <span><h4>Price:</h4><input id="priceInput"></input></span><br>
+  <div>
+    <h4>Tags:</h4>
+    <input type="checkbox" id="dairyTagCheckbox" name="Dairy"/> Dairy <br>
+    <input type="checkbox" id="wheatTagCheckbox" name="Wheat"/> Wheat <br>
+    <input type="checkbox" id="coldCutsTagCheckbox" name="Cold Cuts"/> Cold Cuts <br>
+    <input type="checkbox" id="fruitsAndVegetablesTagCheckbox" name="Friuts And Vegetables"/> Friuts And Vegetables <br>
+  </div>
+  <div>
+    <h4>Stores:</h4>
+    <input type="checkbox" id="stopAndShopStoreCheckbox" name="Stop And Shop"/> Stop And Shop <br>
+    <input type="checkbox" id="targetStoreCheckbox" name="Target"/> Target <br>
+    <input type="checkbox" id="walmartStoreCheckbox" name="Walmart"/> Walmart <br>
+    <input type="checkbox" id="keyFoodStoreCheckbox" name="Key Food"/> Key Food <br>
+  </div>
+  <button class="btn btn-default" onclick="submit()">Submit</button>
+  <table id="productTable" class="table"></table>
+  <script src="https://www.gstatic.com/firebasejs/3.4.0/firebase.js"></script>
+  <script>
+    var allProducts = [];
+
+    class Product{
+      constructor(name, price, tags, stores) {
+        this.name = name;
+        this.price = price;
+        this.tags = tags;
+        this.stores = stores;
+      }
+      addToTable(){
+        $("#productTable").append("<tr><td>" + this.name + "</td><td>" + this.price + "</td><td>" + this.tags + "</td><td>" + this.stores + "</td></tr>");
+      }
+    }
+    var tagCheckboxes = [$("#dairyTagCheckbox"), $("#wheatTagCheckbox"), $("#coldCutsTagCheckbox"), $("#fruitsAndVegetablesTagCheckbox")];
+    var storeCheckboxes = [$("#stopAndShopStoreCheckbox"), $("#targetStoreCheckbox"), $("#walmartStoreCheckbox"), $("#keyFoodStoreCheckbox")];
+
+    function submit(){
+      let tags = [];
```

Comment:
Arrays / Objects can be const. 

```js
const tags = [];
tags.push(1); // this is okay since we never actually go tags = [1]; (this would break)
```